addons/sale_expense/models/account_move_line.py

```diff
@@ -30,28 +34,19 @@ def _sale_determine_order(self):
         return mapping_from_invoice
 
     def _sale_prepare_sale_line_values(self, order, price):
+        # EXTENDS sale
         # Add expense quantity to sales order line and update the sales order price because it will be charged to the customer in the end.
+        # ensure_one() in super
```

Comment:
Might as well put it for real if you put the comment... (which I would argue is useless)